scene/3d/sprite_3d.cpp

```diff
@@ -597,6 +597,15 @@ void Sprite3D::_draw() {
 		RS::get_singleton()->material_set_param(get_material(), "texture_albedo", texture->get_rid());
 		last_texture = texture->get_rid();
 	}
+	if (!normal_map.is_valid() && last_normal_map != RID()) {
+		last_normal_map = RID();
+		RS::get_singleton()->material_set_param(get_material(), "texture_normal", Variant());
+	}
+	if (normal_map.is_valid() && last_normal_map != normal_map->get_rid()) {
+		RS::get_singleton()->material_set_param(get_material(), "normal_scale", 1);
```

Comment:
Should the normal map scale be exposed as a property? Given proposals like https://github.com/godotengine/godot-proposals/issues/3326, I think it'd be better to expose it.

Reply:
Will do!

Reply:
Done